docs/CONTRIBUTING.md

```diff
@@ -222,3 +222,19 @@ extensions.lmem@gmail.com
 
 To generate your own tokens and deploy from your local environment :
 https://github.com/DrewML/chrome-webstore-upload/blob/master/How%20to%20generate%20Google%20API%20keys.md
+
+### Firefox Addons
+
+> [Developer Hub](https://addons.mozilla.org/en-US/developers/addons) 
+>
+> infrastructure@dismoi.io
+
+In order for the Mozilla review to complete successfully, please the following steps :
```

Comment:
Why is this `extension` publishing doc added to the backend repo ?  and to this PR ?